hackathon/urls.py

```diff
@@ -1,9 +1,10 @@
 from django.urls import path
 
-from .views import HackathonListView, create_hackathon, delete_hackathon
+from .views import HackathonListView, create_hackathon, update_hackathon, delete_hackathon
```

Comment:
This line is too long as per PEP8 it should be less than 79 chars. Please make sure that all of the lines/files adhere to this.

Reply:
Done, this will be visible once I push the changes to the existing PR.